.github/workflows/create-release.yml

```diff
@@ -30,14 +30,19 @@ jobs:
         run: |
           git config user.name "API Team CI User"
           git config user.email "aciattestteamci@deque.com"
-      - run: npx standard-version --release-as="${{ github.events.inputs.releaseLevel }}"
+      - id: release
+        name: Release
+        run: |
+          npx standard-version --release-as="${{ github.events.inputs.releaseLevel }}"
+          Version=$(jq -r .version package.json)
+          echo "::set-output name=Version::$Version"
```

Comment:
I suggest that changes to workflows not be buried in a dependency patch release.